pages/quantum.js

```diff
@@ -235,15 +235,15 @@ export default function Quantum() {
               <Card.Footer>Footer of the Card</Card.Footer>
             </Card>
           </div>
-          <div className="MarginTopBottom ComponentsContainer RoundedEdges WhiteBackground">
+          <div className="Margins  RoundedEdges WhiteBackground">
             <h4 className="TextFormattingH4">Alert</h4>
             <h6 className="TextFormattingH6">
               Alerts are used for items that need to be labeled, categorized, or
               organized using keywords.
             </h6>
             <Alert>Alert Test</Alert>
           </div>
-          <div className="MarginTopBottom ComponentsContainer RoundedEdges WhiteBackground">
+          <div className="Margins  RoundedEdges WhiteBackground">
```

Comment:
The accordion example has a different width than the ChargeBar example. The same happens with Autocomplete, Dropdown and others. Can you check if there was a lack of styling?